### PR TITLE
Adjusted the position button, changed save control into a drop down and prefix of the new column in copy Row names dialog

### PR DESCRIPTION
**Pull request by Gioche6**
Fixes issue #6684 

@africanmathsinitiative/developers , this is ready for review

The saving control was set as a text instead of a combo. I have fixed the issue and resized the dialog.

@rdstern , @Patowhiz , This is ready for review

### CODE REVIEW

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern Please can you test? thanks
(or one of the test team could test?)

**Review by rdstern — CHANGES_REQUESTED** [review]
I suggest a trivial change.  The default name is row_names.  Could we just call it row instead.  
 The reason is that the name is currently row_names(C) as it is a character variable.  But you don't see the C, in the spreadsheet.  With the shorter name you will.
I would still prefer row as the default name, but a more complex change could be to make the resulting variable numeric if the contents is the row number - which it usually is!

**Comment by Patowhiz** [comment]
> I suggest a trivial change. The default name is row_names. Could we just call it row instead.
> The reason is that the name is currently row_names(C) as it is a character variable. But you don't see the C, in the spreadsheet. With the shorter name you will.

@rdstern, @Gioche6 will do the above in this PR.

> I would still prefer row as the default name, but a more complex change could be to make the resulting variable numeric if the contents is the row number - which it usually is!

@rdstern do you want us to prioritise the above in this code sprint? 
@Gioche6 create another issue from the above @rdstern comment. I thinks its best for to do it in a separate PR.

**Comment by Patowhiz** [comment]
> I suggest a trivial change. The default name is row_names. Could we just call it row instead.
> The reason is that the name is currently row_names(C) as it is a character variable. But you don't see the C, in the spreadsheet. With the shorter name you will.

@rdstern, @Gioche6 will do the above in this PR.

> I would still prefer row as the default name, but a more complex change could be to make the resulting variable numeric if the contents is the row number - which it usually is!

@rdstern do you want us to prioritise the above in this code sprint? 
@Gioche6 create another issue from the above @rdstern comment. I thinks its best for to do it in a separate PR.

**Review by lloyddewit — APPROVED** [review]

**Comment by lloyddewit** [comment]
@rdstern If you test/approve, then we can merge, thanks

**Review by rdstern — DISMISSED** [review]

**Comment by Gioche6** [comment]
@rdstern  and @lloyddewit , I have fixed the merging conflict error that had appeared . This is ready for reviewing and merging.

**Review by rdstern — APPROVED** [review]

**Comment by Patowhiz** [comment]
@rdstern, this looks good on my side. Ready for your review.

File: instat/dlgRownamesOrNumbers.vb [hunk 1]
```diff
@@ -89,8 +89,8 @@ Public Class dlgRowNamesOrNumbers
         ucrChkAsNumeric.SetRDefault("TRUE")
 
         'ucrNewColumnName
-        ucrNewColumnName.SetIsTextBox()
-        ucrNewColumnName.SetPrefix("row_names")
+        ucrNewColumnName.SetIsComboBox()
+        ucrNewColumnName.SetPrefix("row")
         ucrNewColumnName.SetSaveTypeAsColumn()
         ucrNewColumnName.SetDataFrameSelector(ucrSelectorRowNames.ucrAvailableDataFrames)
         ucrNewColumnName.SetLabelText("Column Name:")
```
Comment (shadrackkibet) [inline instat/dlgRownamesOrNumbers.vb:96]: Something to discuss and agree on and use consistently. Do we use `Column Name:` or `New Column Name:`?

File: instat/dlgRownamesOrNumbers.vb [hunk 2]
```diff
@@ -89,8 +89,8 @@ Public Class dlgRowNamesOrNumbers
         ucrChkAsNumeric.SetRDefault("TRUE")
 
         'ucrNewColumnName
-        ucrNewColumnName.SetIsTextBox()
-        ucrNewColumnName.SetPrefix("row_names")
+        ucrNewColumnName.SetIsComboBox()
+        ucrNewColumnName.SetPrefix("row")
```
Comment (shadrackkibet) [inline instat/dlgRownamesOrNumbers.vb:93]: `row_names` sounded much more appropriate to me.

**Review by shadrackkibet — APPROVED** [review]
Minor comments. Either way, I approve and merge.

**Pull request merged**